tools/autograd/templates/python_fft_functions.cpp

```diff
@@ -5,7 +5,7 @@
 #include "torch/csrc/DynamicTypes.h"
 #include "torch/csrc/Exceptions.h"
 #include "torch/csrc/autograd/python_fft_functions.h"
-#include "torch/csrc/autograd/python_return_types.h"
+#include "torch/csrc/autograd/generated/python_return_types.h"
```

Comment:
Could you send an example of the generated code? Perhaps in a gist?

Reply:
Here are the entire generated files content: https://gist.github.com/andreigh/b1b282c0ba40666a979457057fe94bb2